static/src/javascripts-legacy/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -19,7 +21,9 @@ define([
     regularsV2,
     acquisitionsEpicArticle50Trigger,
     acquisitionsEpicDesignVariationsV2,
-    laundromat
+    laundromat,
+    acquisitionsEpicVsEpicAndEngagementBanner,
+    reduce
```

Comment:
can this be moved to the top of the import list, so we can separate libraries from imported tests  ??